### PR TITLE
Docs: Create gauge-panel.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
Adds updated Gauge visualization doc for 7.0

### CODE REVIEW

File: docs/sources/panels/visualizations/gauge-panel.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- Field overrides
+- [Thresholds]({{< relref "../thresholds.md">}})
+- Value mappings
+
+## Display options
+
+Use the following options to refine your visualization:
+
+- **Show -** Choose how Grafana displays your data.
+  - **Calculate -** Show a calculated value based on all rows.
```
Comment (dprokop) [inline docs/sources/panels/visualizations/gauge-panel.md:31]: Would it be a good idea to add link to calculations list here?

File: docs/sources/panels/visualizations/gauge-panel.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  - **All values -** Show a separate stat for every row. If you select this option, then you can also select a **Limit**, or the maximum number of rows to display.
+- **Value -** Select a reducer function that Grafana will use to reduce many fields to a single value. Click the **Value** list to see functions and brief descriptions.
+- **Orientation -** Choose a stacking direction.
+  - **Auto -** Grafana selects what it thinks is the best orientation.
```
Comment (dprokop) [inline docs/sources/panels/visualizations/gauge-panel.md:35]: Think here's something wrong with the hierarchy.

When you select `Calculate` you will get the `Value` option.
When you select `All value` you will get Limit option

Value is duplicated ^^ and indented unnecessary (the first occurrence)

**Review by dprokop — APPROVED** [review]
Small nit, other then that 🔥

File: docs/sources/panels/visualizations/gauge-panel.md (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/panels/visualizations/gauge-panel.md:31]: Yes!

**Pull request merged**